README.md

```diff
@@ -1,5 +1,3 @@
-# FCC Test Suite Template
+# Review your project from trials!
 
-**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
-
-You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
+This repo is created just to learn how to review a code
```

Comment:
Good to have added some content into your README.md. But I think it's not so clear to be understandable. Add more information such as what has been done in the project and how to run it locally